setuptools.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: setuptools
+  version: 69.2.0
+  epoch: 1
+  description: Easily download, build, install, upgrade, and uninstall Python packages
+  copyright:
+    - license: "MIT"
+  dependencies:
+    provides:
+      - whl:setuptools=${{package.version}}
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-pip
+      - py3-wheel
+      - python-3.10-base
+      - python-3.11-base
+      - python-3.12-base
+      - wolfi-base
+
+data:
+  - name: py-versions
+    items:
+      3.12: 312
+      3.11: 311
+      3.10: 310
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: 0ff4183f8f42cd8fa3acea16c45205521a4ef28f73c6391d8a25e92893134f2e
+      uri: https://files.pythonhosted.org/packages/source/s/setuptools/setuptools-${{package.version}}.tar.gz
+
+  - name: Build Python Wheels
+    runs: |
+      pip wheel --no-deps .
+      mkdir -p ${{targets.destdir}}/usr/share/python-wheel/
+      mv *.whl ${{targets.destdir}}/usr/share/python-wheel/
+
+subpackages:
+  - range: py-versions
```

Comment:
i dont see how, but I'd really like to define the range based on the 'pythons-meta' or something. but that feels ilke too much magic at this point.

Reply:
Ideally we would do `include yaml`